content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -0,0 +1,112 @@
+===============================================================
+Use Windows IoT software to connect devices to an Odoo database
+===============================================================
+
+A Virtual IoT box is a computer program that needs to be downloaded and installed on a Windows
+computer. This requires a Windows operating system with an Odoo 16 or later database.
+
+The Windows Virtual :abbr:`IoT (Internet of Things)` box works the same way as a physical
+:abbr:`IoT (Internet of Things)` box, with the ability to run most of the same devices. All
+:abbr:`POS (Point of Sale)` devices work with it, such as a scale or printer. Payment terminals will
+also work, but it should be noted that :abbr:`MRP (Material Requirement Planning)` devices are not
+compatible. These include cameras or measurement tools.
+
+Pre-requisites
+==============
+
+- Odoo 16 database
+- :abbr:`IoT (Internet of Things)` compatible devices
+- Device drivers for Windows.
+- Windows computer (laptop, desktop, server)
+
+Connect the Windows virtual Iot box to an Odoo database
+=======================================================
+
+#. Navigate to the Odoo 16 or higher installation package for Enterprise/Community Windows at
+   `Odoo's Download Page <https://odoo.com/download>`_.
+
+#. Install and setup the Odoo .exe file.
+
+   After the instructions screen click :guilabel:`Next` to start the installation. Agree to the
+   :abbr:`TOS (Terms of Service)`.
+
+#. During the next step of the installation, select :guilabel:`Odoo IoT` from the :guilabel:`Select
+   the type of install` dropdown.
+
+   For reference, the following should be installed:
+
+   - **Odoo server**
+   - **Odoo IoT**
+   - **Nginx WebServer**
+   - **Ghostscript interpreter**
+
+
+   Ensure there is enough space on the computer for the installation and click :guilabel:`Next`.
+
+#. Select the :guilabel:`Destination Folder`.
+
+   .. warning::
+      Odoo's Windows virtual IoT software shouldn't be installed inside any of the Window's User's
+      directories. Doing so won't allow for Nginx to initialize.
+
+   Choosing ``C:\odoo`` as the install location will allow for the Nginx server to start. Click
```

Comment:
don't know if it will allow you if the "odoo" folder does not exists in C:. Can you confirm to me that it will create the folder if it does not exists ?